submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+            </div>
+            <div class="wrapper_form-card_number">
+              <input
+                type="text"
+                class="form-sign_pay_method_inputs  input_card_number validate_on_focus"
+                placeholder="0000 0000 0000 0000"
+                minlength="16"
+                maxlength="16"
+              />
+              <span class="input_card_data_title card_number_title"></span>
+              <input
+                type="datetime"
+                class=" form-sign_pay_method_inputs  input_date validate_on_focus"
+                placeholder="DD/MM/YY"
+                maxlength="6"
+                minlength="6"
+              />
+              <span class="input_card_data_title input_date_title"></span>
+              <input
+                type="text"
+                class="form-sign_pay_method_inputs  input_code validate_on_focus"
+                inputmode="numeric"
+                placeholder="000"
+                minlength="3"
+                maxlength="3"
+              />
+              <span class="input_card_data_title input_card_code_title"></span>
+            </div>
+          </section>
+        </article>
+        <section class="form-submit">
+          <div class="wrapper_agree_checkbox">
+            <input
+              type="checkbox"
+              name=""
+              id="agree"
+              class="visually__hidden"
+              required
+            />
+            <span class="custom_checkbox"></span>
+            <label for="agree" class="form-label_agree">
+              I agree to the
+              <a href="#" class="submit_link">
+                Terms & Conditions
+              </a>
+            </label>
+          </div>
+          <input
+            type="submit"
+            value="place order"
+            class="button submit_button form-submit_input"
+          />
+          <input
+            type="button"
+            value="print a quote"
+            class="button form-submit_input form-header_print_button"
+          />
+        </section>
+      </form>
+    </main>
+    <footer class="form-footer">
+      <section class=" form-footer_block form-footer_info">
+        <span class="form-footer_info_header">What we do</span>
+        <p class="form-footer_info_text">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Nemo nesciunt
+          commodi suscipit hic deserunt dolorum provident, consectetur amet,
+          quos vel maxime atque voluptas? Rerum reiciendis quod eligendi,
+          corporis labore minus.
+        </p>
+      </section>
+      <section class="form-footer_block form-footer_adress">
+        <span class="form-footer_info_header office_title">
+          Our main office
+        </span>
+        <ul class="form-footer_list">
+          <li class="form-footer_li">Dubai,UAE</li>
+          <li class="form-footer_li">P.O.Box : 502735</li>
+          <li class="form-footer_li">Phone:+971(0) 44278128</li>
```

Comment:
Use a link tag for this as on the form header.

Reply:
Done.